### PR TITLE
CVE-2014-7941 and CVE-2011-5319

### PR DESCRIPTION
**Pull request by wylaris**
Creating a PR as per round 1 instruction 9 for CVE-2014-7941 and CVE-2011-5319.

### CODE REVIEW

**Comment by wylaris** [comment]
All tests pass, does not like having the beginning of a git sha be capitalized.

File: cves/CVE-2011-5319.yml [hunk 1]
```diff
@@ -93,20 +96,20 @@ discovered:
...
 subsystem:
   question: |
     What subsystems was the mistake in?
 
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged. Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer: 
-  name: 
+  answer: Based on the path of the changed file
```
Comment (DanTheManGude) [inline cves/CVE-2011-5319.yml]: Content sounds vague, maybe consider adding the full path or be more descriptive than just saying it was from the path.

File: cves/CVE-2014-7941.yml [hunk 2]
```diff
@@ -52,10 +56,10 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - :commit: f09a3116b3c3bff8d4c98dd65d659471a7eeff6f
-  :note: ''
+  :note: 'Pass in long to XChangeProperty() instead of an int when using format=32'
```
Comment (DanTheManGude) [inline cves/CVE-2014-7941.yml]: Was this commit fixing a bug of passing a long in or adding new functionality that led to the bug?  How does this commit relate?

**Comment by DanTheManGude** [comment]
Upvotes – 5 – CVE-2011-5319
Upvotes – 1 – CVE-2014-7941

File: cves/CVE-2011-5319.yml [hunk 3]
```diff
@@ -33,6 +33,9 @@ description_instructions: |
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
 description:
```
[the remark this answers is not in the document]
Comment (wylaris) [reply cves/CVE-2011-5319.yml]: Separated and rewrote the sentences while explaining what they keystrokes are for.

File: cves/CVE-2011-5319.yml [hunk 4]
```diff
@@ -45,16 +48,16 @@ reviews:
...
 - 421691
-repo: 
+repo: https://chromium.googlesource.com/chromium
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - :commit: d771a782774564197c9c4d3ddd07605e53397a12
-  :note: ''
+  :note: 'Add a unit test for DeviceMotionEventPump to ensure that events are not delivered to Blink at a rate faster than 60 Hz.'
```
[the remark this answers is not in the document]
Comment (wylaris) [reply cves/CVE-2011-5319.yml]: Explained what DeviceMotionEventPump and Blink are.

File: cves/CVE-2011-5319.yml [hunk 5]
```diff
@@ -45,16 +48,16 @@ reviews:
...
 vccs:
-- :commit:
-  :note: 
+- :commit: 3cc4db828f71ceb077a6093073f3bfbf449f92ee
+  :note: 'Rename device_orientation to device_sensors. This change is needed to add DeviceLight api.'
```
[the remark this answers is not in the document]
Comment (wylaris) [reply cves/CVE-2011-5319.yml]: Added an explanation of what DeviceLight API does.

File: cves/CVE-2011-5319.yml (hunk 1 above)
Comment (wylaris) [reply cves/CVE-2011-5319.yml]: This is what was given as an example but I added the file path for clarity.

File: cves/CVE-2011-5319.yml [hunk 6]
```diff
@@ -45,16 +48,16 @@ reviews:
...
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
```
[the remark this answers is not in the document]
Comment (wylaris) [reply cves/CVE-2011-5319.yml]: Added upvotes to file.

File: cves/CVE-2014-7941.yml (hunk 2 above)
Comment (wylaris) [reply cves/CVE-2014-7941.yml]: Added a clarification as to what the commit was fixing.

**Pull request merged**